res/values/strings.xml

```diff
@@ -125,6 +125,9 @@
   <string name="ui_mark_as_read_label">Mark as read (emails)</string>
   <string name="ui_mark_as_read_desc">Whether to mark messages in Gmail as read or not.</string>
 
+  <string name="ui_sync_read_flag_label">Save read flag</string>
+  <string name="ui_sync_read_flag_desc">Whether to mark unread SMS and MMS in Gmail as unread or not.</string>
```

Comment:
maybe there's a way to combine the two settings into one? that would make it less confusing (always mark as read, always mark as unread, use message status) or something like that